website/pages/docs/job-specification/proxy.mdx

```diff
@@ -19,42 +19,46 @@ Connect](/docs/integrations/consul-connect). It is valid only
 within the context of a `sidecar_service` stanza.
 
 ```hcl
- job "countdash" {
-   datacenters = ["dc1"]
-   group "api" {
-     network {
-       mode = "bridge"
-     }
+job "countdash" {
+  datacenters = ["dc1"]
+  group "api" {
+    network {
+      mode = "bridge"
+    }
 
-     service {
-       name = "count-api"
-       port = "9001"
-
-       connect {
-         sidecar_service {}
-       }
-     }
-     task "web" {
-         driver = "docker"
-         config {
-           image = "test/test:v1"
-         }
-     }
-   }
- }
+    service {
+      name = "count-api"
+      port = "9001"
 
+      connect {
+        sidecar_service {
+          proxy {}
+        }
+      }
+    }
+    task "web" {
+      driver = "docker"
+      config {
+        image = "test/test:v1"
+      }
+    }
+  }
+}
 ```
 
 ## `proxy` Parameters
 
 - `local_service_address` `(string: "127.0.0.1")` - The address the local service binds to. Useful to
   customize in clusters with mixed Connect and non-Connect services.
-- `local_service_port` <code>(int:[port][])</code> - The port the local service binds to.
+- `local_service_port` `(int:[port][])` - The port the local service binds to.
   Usually the same as the parent service's port, it is useful to customize in clusters with mixed
   Connect and non-Connect services
 - `upstreams` <code>([upstreams][]: nil)</code> - Used to configure details of each upstream service that
   this sidecar proxy communicates with.
-- `config` <code>(map: nil)</code> - Proxy configuration that's opaque to Nomad and
+- `expose` <code>([expose]: nil)</code> - Used to configure expose path configuration for Envoy.
```

Comment:
```suggestion
- `expose` `([expose]: nil)` - Used to configure expose path configuration for Envoy.
```

Reply:
Without the `<code>` tags, the `expose` name isn't rendered as a link to the expose doc page like we want it to. Similar story with `upstreams` above. I think it has something to do with the brackets, but /shrug